components/hub75_matrix_display/matrix_display.cpp

```diff
@@ -14,17 +14,30 @@ namespace esphome
         {
             ESP_LOGCONFIG(TAG, "Setting up MatrixDisplay...");
 
-            // The min refresh rate correlates with the update frequency of the component
-            this->mxconfig_.min_refresh_rate = 1000 / update_interval_;
-
-            this->mxconfig_.double_buff = true;
+            // Handle "never" and 0 to avoid divide by 0
+            if (this->update_interval_ == 4294967295 || this->update_interval_ == 0) {
```

Comment:
what does this magic number mean and can we get rid of it (or write it differently)?

Non-LVGL users should still be able to disable the update interval for manual operation.